.github/workflows/deploy.yml

```diff
@@ -92,3 +92,30 @@ jobs:
           no-cache: true
           build-args: DISTBALL=dist/target/camunda-zeebe-*-SNAPSHOT.tar.gz
           target: app
+          
+  notify-if-failed:
```

Comment:
:wrench: Would be nice to add a name to the job here, something like "Notify Slack on build failure"

Reply:
Ah yeah I always expect that it takes the name which I defined here